js/packages/openinference-instrumentation-beeai/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": ".",
+    "types": ["node", "jest"],
+    "skipLibCheck": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext"
```

Comment:
Overriding these from each of the base tsconfig files defeats the purpose of building esm + commonjs bundles.

target, module, and moduleResolution should not be declared in your tsconfigs so that we can maintain build parity with the other instrumentation by inheriting these fields from our base tsconfigs